src/RGBPatternManager.cpp

```diff
@@ -97,12 +97,21 @@ void RGBPatternManager::RunPattern(void* arg) {
 
   while (true) {
     for (const auto& state : pattern) {
+      // WS2812B usually takes commands in GRB order
+      // https://cdn-shop.adafruit.com/datasheets/WS2812B.pdf - Page 5
+      // But some actually expect RGB!
+
+#if defined(OPENSHOCK_LED_FLIP_RG_CHANNELS) && OPENSHOCK_LED_FLIP_RG_CHANNELS
```

Comment:
For a value where you only check if its truthy or falsey, you dont need to check if its defined first, you can just directly do ``#if OPENSHOCK_LED_FLIP_RG_CHANNELS``